var/spack/repos/builtin/packages/dealii/package.py

```diff
@@ -289,6 +296,13 @@ class Dealii(CMakePackage, CudaPackage):
               msg='The interface to Nanoflann was removed from version 9.3.0. '
                   'Please explicitly disable this variant via ~nanoflann')
 
+    # interfaces added in 9.4.0:
+    for p in ['cgal']:
+        conflicts('+{0}'.format(p), when='@:9.3',
+                  msg='The interface to {0} is supported from version 9.4.0 '
+                      'onwards. Please explicitly disable this variant '
+                      'via ~{0}'.format(p))
```

Comment:
Can you remove this conflict and use a conditional variant?

Reply:
Done. Thank you!